sites/melbourne/app/models/melbourne/event/schema.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# https://developers.google.com/search/docs/appearance/structured-data/event
+module Melbourne
+  class Event
+    private_constant :Schema
+    class Schema
+      TYPE = "Event"
+
+      def initialize(event)
+        @event = event
+      end
+
+      def to_h # rubocop:disable Metrics/MethodLength
```

Comment:
Same as above; seemed a bit excessive but I can change it if anyone feels strongly